WcaOnRails/spec/models/user_spec.rb

```diff
@@ -345,7 +345,8 @@
       other_person = FactoryGirl.create :person, year: 1980, month: 2, day: 1
       user.unconfirmed_wca_id = other_person.wca_id
       expect(user).to be_invalid
-      expect(user.errors.messages[:dob_verification]).to eq ['incorrect']
+      expect(user.errors.messages[:dob_verification].length).to eq 1
+      expect(user.errors.messages[:dob_verification][0]).to start_with "The provided date of birth doesn't match our database."
```

Comment:
Hm... The much better way is to expect the translation key (`"dob_incorrect_html"`). As I remember it, it's neither trivial or very hard to do, but I don't remember how we did it :)

It's really worth having in an internationalized project, but is probably for a separate PR.